.aws/src/event-rules/user-registration/userRegistrationEventRule.ts

```diff
@@ -0,0 +1,116 @@
+import { Construct } from 'constructs';
+import { Resource } from 'cdktf';
+import {
+  PocketEventBridgeProps,
+  PocketEventBridgeRuleWithMultipleTargets,
+  ApplicationEventBus,
+  PocketPagerDuty,
+} from '@pocket-tools/terraform-modules';
+import { config } from '../../config';
+import { iam, sns, sqs } from '@cdktf/provider-aws';
+import { eventConfig } from './eventConfig';
+import { createDeadLetterQueueAlarm } from '../utils';
+import * as NullProviders from '@cdktf/provider-null';
+
+export class UserRegistrationEventRule extends Resource {
+  public readonly snsTopic: sns.SnsTopic;
+  public readonly snsTopicDlq: sqs.SqsQueue;
+
+  constructor(
+    scope: Construct,
+    private name: string,
+    private pagerDuty: PocketPagerDuty
+  ) {
+    super(scope, name);
+
+    this.snsTopic = new sns.SnsTopic(this, 'user-registration-topic', {
+      name: `${config.prefix}-UserRegistrationTopic`,
+      lifecycle: {
+        preventDestroy: true,
+      },
+    });
+
+    this.snsTopicDlq = new sqs.SqsQueue(this, 'sns-topic-dql', {
+      name: `${config.prefix}-SNS-Topic-Event-Rule-DLQ`,
```

Comment:
should this reference user registration? like `${config.prefix}-UserRegistrationTopic-DLQ`?

Reply:
thanks for catching that @jpetto  :) 
looks like c-p error, we have similar names in other events too - updated them all